great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -63,7 +64,7 @@
     "Snowflake",
     "Databricks (SQL)",
 ]
-DATA_QUALITY_ISSUES = ["Sets"]
+DATA_QUALITY_ISSUES = [DataQualityIssues.UNIQUENESS.value]
```

Comment:
I'm not a fan of applying the enum value here, but there seem to be some nuances when applying the actual value to the docstrings since it's meant to be static. I'm open to suggestions, but this seems like the cheapest solution that I can think of.

Reply:
It does end up being a little verbose, but it hopefully will be a thing we don't have to mess with much!